x-pack/plugins/security_solution/public/timelines/components/timeline/properties/helpers.tsx

```diff
@@ -104,55 +130,153 @@ interface DescriptionProps {
   description: string;
   timelineId: string;
   updateDescription: UpdateDescription;
+  isTextArea?: boolean;
+  disableAutoSave?: boolean;
+  disableTooltip?: boolean;
+  disabled?: boolean;
+  marginRight?: number;
 }
 
 export const Description = React.memo<DescriptionProps>(
-  ({ description, timelineId, updateDescription }) => (
-    <EuiToolTip data-test-subj="timeline-description-tool-tip" content={i18n.DESCRIPTION_TOOL_TIP}>
-      <DescriptionContainer data-test-subj="description-container">
-        <EuiFieldText
-          aria-label={i18n.TIMELINE_DESCRIPTION}
-          data-test-subj="timeline-description"
-          fullWidth={true}
-          onChange={(e) => updateDescription({ id: timelineId, description: e.target.value })}
-          placeholder={i18n.DESCRIPTION}
-          spellCheck={true}
-          value={description}
-        />
+  ({
+    description,
+    timelineId,
+    updateDescription,
+    isTextArea = false,
+    disableAutoSave = false,
+    disableTooltip = false,
+    disabled = false,
+    marginRight,
+  }) => {
+    const onDescriptionChanged = useCallback(
+      (e) => {
+        updateDescription({ id: timelineId, description: e.target.value, disableAutoSave });
```

Comment:
maybe we should add `debounce` to don't dispatch `updateDescription` on every key press?

Reply:
It seems that there's already a delay 500ms in x-pack/plugins/security_solution/public/timelines/store/timeline/epic.ts Line 500, so I think it's ok here